cylc/flow/task_events_mgr.py

```diff
@@ -742,7 +742,7 @@ def process_message(
                 # Already failed.
                 return True
             if self._process_message_failed(
-                itask, event_time, self.JOB_FAILED, forced
+                itask, event_time, self.JOB_FAILED, forced, message
```

Comment:
No need for this as this will make the log message
```
[1/foo:waiting] => failed with failed
```

```suggestion
                itask, event_time, self.JOB_FAILED, forced
```

Reply:
No, it's failed with reason:
```
[1/foo:waiting] => failed with failed/XTERM
```

Reply:
Not in this case, it is `self.EVENT_FAILED` which is `'failed'`

Reply:
(Duplication resolved by changing the logged message)